string2.py

```diff
@@ -18,8 +18,12 @@
 
 
 def verbing(s):
-    """Your code goes here.  Edit this docstring."""
-    return
+    if len(s) >= 3 and not s.endswith('ing'):
+        s += 'ing'
+    elif 'ing' in s:
```

Comment:
Careful here: This will test if 'ing' is _anywhere_ in s, not just at the end of the string like the instructions say.